src/chains.jl

```diff
@@ -95,6 +104,11 @@ function namesingroup(chains::Chains, sym::Symbol)
     return names_of_params[indices]
 end
 
+"""
+    group(chains::Chains, name::Union{String,Symbol}
+
+Returns a subset of the chain chain with all parameters in the group `:name`.
```

Comment:
Though probably

```suggestion
Return a subset of the chain chain with all parameters in the group `Symbol(name)`.
```
is better here.